posthog/queries/person_query.py

```diff
@@ -319,3 +321,11 @@ def _get_email_clause(self) -> Tuple[str, Dict]:
                 Property(key="email", value=self._filter.email, type="person"), 0, prepend="_email"
             )
         return "", {}
+
+    def _get_updated_after_clause(self) -> Tuple[str, Dict]:
+        if not isinstance(self._filter, Filter):
+            return "", {}
+
+        if self._filter.updated_after:
+            return "and max(_timestamp) > toDateTime(%(updated_after)s)", {"updated_after": self._filter.updated_after}
```

Comment:
question: Should we use `parseDateTimeBestEffort` instead of `toDateTime` to support ISO 8601 dates? I believe HogQL also relies on `parseDateTimeBestEffort`.

Reply:
good idea